playbooks/roles/githubsshkeys/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: install httplib2
+  sudo: true
+  pip: name=httplib2
+
+- name: get SSH keys from GitHub
+  uri: >
+    url=https://github.com/{{ item }}.keys
+    return_content=true
+  register: ssh_keys
+  with_items: add_ssh_keys_from_github.usernames
+
+- name: add SSH key labels to ~/.ssh/authorized_keys
+  lineinfile: >
+    dest=~/.ssh/authorized_keys
+    line="# {{ item }}"
+  with_items: add_ssh_keys_from_github.usernames
+
+- name: add SSH keys to ~/.ssh/authorized_keys
+  lineinfile: >
+    dest=~/.ssh/authorized_keys
+    line="{{ item.0.content }}"
```

Comment:
@mrgnr yea we had some users with their desktop ssh and laptop ssh and it added their both keys